.github/workflows/ci.yml

```diff
@@ -3,10 +3,7 @@ on:
   push:
     branches:
       - master
-      - release-[6-9].*
-  pull_request:
```

Comment:
This is why CI didn't run. You removed the pull request trigger!

Reply:
Ah